cloudinit/config/cc_apt_configure.py

```diff
@@ -555,20 +555,50 @@ def is_deb822_format(apt_src_content: str) -> bool:
 
 DEFAULT_APT_CFG = {
     "sourcelist": "/etc/apt/sources.list",
-    "sourceparts": "/etc/apt/sources.list.d",
+    "sourceparts": "/etc/apt/sources.list.d/",
 }
 
+APT_CFG_RE = (
+    r"(Dir::Etc|Dir::Etc::sourceparts|Dir::Etc::sourcelist) \"([^\"]+)"
+)
+
 
 def get_apt_cfg() -> Dict[str, str]:
-    """Return a dict of applicable apt configuration or defaults."""
-    return {
-        "sourcelist": apt_pkg.config.find_file(
```

Comment:
The import for apt_pkg still exists at 39c5c06c9e92, which conflicts with the commit message. Please fix this if you want this to be a rebase commit.

Also it feels a bit odd to introduce code (the import for example) in one commit in a series, only to relocate / refactor the same code in a followup commit in the serious. Not a huge issue in this case, since in this case (as you mentioned) it makes the review easier due to the size of this PR, but I think that's something we want to avoid when we can for rebase commits.

Reply:
Agreed. If I had stuck with the desire to rebase merge the PR this commit set would have needed tidying up.